Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/jee/jsf/UserAuthorizationListener.java

```diff
@@ -154,4 +154,26 @@ private boolean checkConfiguredDisplay(String currentPage) {
         return true;
     }
 
+    /*Cross site Request Forgery Fortify issue can be mitigated via sending tokens for each .xhtml page and validating them.
```

Comment:
It feels like most of this documentation belongs on the Fortify mitigation page; the rest, can you turn into an actual Javadoc comment?